code/game/gamemodes/game_mode.dm

```diff
@@ -488,7 +488,7 @@
 			text += " <span class='boldannounce'>died</span>"
 		else
 			text += " <span class='greenannounce'>survived</span>"
-		if(fleecheck && ply.current.z > ZLEVEL_STATION)
+		if(fleecheck && ply.current.z > ZLEVEL_STATION_PRIMARY)

```

Comment:
This doesn't seem right for this context. Should work on all station z levels no?

Reply:
Most of the use of the define is just autoreplace to put off having to refactor so many systems in a single PR/make sure it still works in the man time